app/battle/battle.py

```diff
@@ -0,0 +1,19 @@
+class Battle:
+
+    def __init__(self, knight_1, knight_2):
+        self.knight_1 = knight_1
+        self.knight_2 = knight_2
+
+    def start_battle(self):
+        while True:
```

Comment:
In battle each knight does one hit each other, not till someone dies.

Reply:
Thanks for the comment, but in my case they are fighting to the end and only one will win